go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/fatih/color v1.10.0
 	github.com/ghodss/yaml v1.0.0
 	github.com/giantswarm/apiextensions/v3 v3.22.0
-	github.com/giantswarm/app/v4 v4.12.0
+	github.com/giantswarm/app/v4 v4.11.0
```

Comment:
I think I broke this as I had to downgrade or I got compile errors.

There is nothing relevant we need in the 4.12.0 release.